docs/en/source/_build/html/_sources/onepageTutorial.rst.txt

```diff
@@ -0,0 +1,327 @@
+5. Your Kafka will run on default port 9092 and connect to ZooKeeper’s default port, 2181.
+
+
+*****************************************
+Running a Kafka Server
+*****************************************
+
+Important: Please ensure that your ZooKeeper instance is up and running before starting a Kafka server.
+
+1. Go to your Kafka installation directory:** C:\kafka_2.11-0.9.0.0\**
+2. Open a command prompt here by pressing Shift + right click and choose the “Open command window here” option).
+3. Now type **.\bin\windows\kafka-server-start.bat .\config\server.properties** and press Enter.
+
+ **.\bin\windows\kafka-server-start.bat .\config\server.properties**
+
+*****************************************
+Setting up PostgreSQL
+*****************************************
+
+Step 1) Go to https://www.postgresql.org/download and select O.S. ,it's Windows for me.
+
+
+Step 2) You are given two options:-
+
+ 1. Interactive Installer by EnterpriseDB
+ 2. Graphical Installer by BigSQL
+
+BigSQL currently installs pgAdmin version 3 which is deprecated. It's best to choose EnterpriseDB which installs the latest version 4
+
+
+Step 3)
+
+ 1.You will be prompted to desired Postgre version and operating system. Select the latest 
+ Postgre version and OS as per your environment
+
+ 2.Click the Download Button
+
+Download will begin
+
+
+Step 4) Open the downloaded exe and Click next on the install welcome screen.
+
+
+Step 5) 
+
+ 1.Change the Installation directory if required, else leave it to default
+
+ 2.Click Next
+
+
+Step 6)
+
+ 1.You may choose the components you want to install in your system. You may uncheck Stack Builder
+
+ 2.Click Next
+
+
+Step 7)
+
+ 1.You may change the data location
+
+ 2.Click Next
+
+
+Step 8)
+
+ 1.Enter super user password. Make a note of it
+
+ 2.Click Next
+
+
+Step 9)
+
+ 1.Leave the port number as default
+
+ 2.Click Next
+
+
+Step 10)
+
+ 1.Check the pre-installation summary:
+ 2.Click Next
+
+
+
+Step 11) Click the next button
+
+
+
+Step 12) Once install is complete you will see the Stack Builder prompt
+
+ 1.Uncheck that option. We will use Stack Builder in more advance tutorials
+
+ 2.Click Finish
+
+
+
+
+Step 13) To launch Postgre go to Start Menu and search pgAdmin 4
+
+
+
+Step 14) You will see pgAdmin homepage
+
+
+
+Step 15) Click on Servers > Postgre SQL 10 in the left tree
+
+Step 16)
+
+ 1.Enter super user password set during installation
+
+ 2.Click OK
+
+Step 17) You will see the Dashboard
+
+
+That's it to Postgre SQL installation.
+
+
+*****************************************
+configure postgresql for the first time
+*****************************************
+ 
+C:\Program Files (x86)\PostgreSQL\10\data
+
+1.Edit the pg_hba.conf file :
+
```

Comment:
Change this!
You are mixing windows and linux here

Reply:
section removed.